utils/members_utils.go

```
@@ -0,0 +1,84 @@
+	for i, member := range members {
+		mentions[i] = fmt.Sprintf("<@%s>", member.User.ID)
+	}
+	return mentions
+}
+
+func FormatRoleMention(roleID string) string {
+	return fmt.Sprintf("<@&%s>", roleID)
+}
+
+func JoinMentions(mentions []string, separator string) string {
+	if mentions == nil || len(mentions) == 0 {
```

golangci-lint (1.64.8)

S1009: should omit nil check; len() for nil slices is defined as zero

(gosimple)